h3/h3.py

```diff
@@ -600,8 +601,9 @@ def hex_range_distances(h3_address, ring_size):
     for i in range(0, ring_size + 1):
         out.append(set([]))
     for i in range(0, array_len):
-        ring_index = distances[i]
-        out[ring_index].add(h3_to_string(krings[i]))
+        if krings[i] != 0:
```

Comment:
I don't believe this is needed because hexRangeDistances would return a failure code instead.

Reply:
> I don't believe this is needed because hexRangeDistances would return a failure code instead.

I see. revert hex_range_distances change.